src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj

```diff
@@ -5,7 +5,7 @@
     <Description>Provides management capabilities for Microsoft Azure IotHub.</Description>
     <AssemblyTitle>Microsoft Azure IotHub Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-    <VersionPrefix>1.1.2</VersionPrefix>
+    <VersionPrefix>1.1.3</VersionPrefix>
     <PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
```

Comment:
@anusapan can you add package release notes like below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10

Reply:
added PackageReleaseNotes.